lib/app/services/converters/string_extension.dart

```diff
@@ -39,3 +39,7 @@ extension StringExtension on String {
     return TextSpan(children: spans);
   }
 }
+
+extension NullableStringExtension on String? {
```

Comment:
Did I first peruse the internet for dart documentation on what .safe might mean BEFORE I scrolled down to see this extension function? Maybe.